CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 0.13.2-dev
+## 0.13.3-dev
+
+## 0.13.2
+
+* Fix `CRLF` in `BaseRequest`
```

Comment:
This could use some context or explanation (`CRLF` is not defined).
Maybe:

 * Validate the `method` parameter of `BaseRequest`. Must now be a valid "token".

This is correct. It doesn't say why, but validation is generally a good thing.

Reply:
fixed as per suggestion, Thank you for all hints!